src/app/common/controllers/JobCtrl.js

```diff
@@ -195,14 +185,15 @@
         .controller('ViewJobCtrl', ['datastoreService', 'commentService', 'jobService', '$scope', '$routeParams',
             function (datastoreService, commentService, jobService, $scope, $routeParams) {
 
-                datastoreService.fetch('jobs/' + $routeParams.id + '.json?include=owner,company,hourly-pay')
-                    .then(function (data) {
-                        var job = data.store.find('jobs', $routeParams.id);
-                        job.max_rate = job["hourly-pay"].rate;
-                        job.totalRate = job.hours * job.max_rate;
-
-                        $scope.job = job;
-                    });
+                $scope.job = jobService.getJob($routeParams.id);
+                $scope.job.$promise.then(function (result) {
+                    $scope.job = result.data;
+                    $scope.job.owner = result.included[0];
+                    $scope.job.company = result.included[1];
+                    $scope.job.max_rate = result.included[2].attributes.rate;
+                    $scope.job.totalRate = $scope.job.attributes.hours * $scope.job.max_rate;
```

Comment:
`$scope.job.max_rate` will not work any more it has been removed from Jobs. Instead job has a relation to HourlyPay, which can be inlined when getting jobs: `/jobs?include=hourly-pay`